### ISSUE TITLE
Awesome!! Minor docs suggestion

### ISSUE THREAD

**Issue by mattzollinhofer** [rails-lambda/lamby-cookiecutter#35]
**Thanks for Putting This Out There!!!**
I used your lamby quickstart and it was amazing -- I have it running! I've been a Rails dev on AWS for a long time and have been hoping to play with Lambda and Rails but haven't had the reason to spend the time. I haven't spent any time with Docker so that was an experiment I was interested to embrace as well. Then I came across lamby and I got everything at once! I figured it's such a quick experiment why not give it a go. Your 5 minutes got me!

_Docs Suggestion_
The only negative in my story is that it wasn't 5 minutes, it was more like a number of hours. 

**tldr;** The `docker-compose.yml` specifies `linux/amd64` which seemed to conflict with the docs saying everything is geared for ARM processing. Launching VS Code and running the dev container threw an error on my Apple M1 Laptop. I _think_ it should be `platform: linux/arm64`.

#### More Details
Being such a newbie to docker specifically got me. The docs say: 

> Our starter defaults to an arm64 deployment target. If needed, see our [CPU Architecture](https://lamby.custominktech.com/docs/cpu) guide on how to switch to x86_64.

This misled me a bit because when I opened up the `my_awesome_lambda` project in VSCode and it attempted to "Start the Dev Container" it ended up failing with this error:

> failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: pull access denied, repository does not exist or may require authorization: server message: insufficient scope: authorization failed

![CleanShot 2023-03-04 at 12 16 10@2x](https://user-images.githubusercontent.com/444766/222919880-66a6fead-4f45-4b8f-befd-d1170e0220d5.png)

## My Fix
I needed to modify the `docker-compose.yml` file to `platform: linux/arm64` from `platform: linux/amd64`. AMD and ARM are soooo close when it's late at night!

I was going to open a PR for you, but I couldn't find the docs repo. Let me know if I can help, or if I just have a fundamental misunderstanding about how this should have worked.

**Comment by metaskills** [comment]
> Then I came across lamby and I got everything at once! I figured it's such a quick experiment why not give it a go. Your 5 minutes got me!

Thank you! Means a lot to me.

> The only negative in my story is that it wasn't 5 minutes, it was more like a number of hours.

Yea, not totally truthful advertising. Especially when it has bugs. Let's work thru those.

> The docker-compose.yml specifies linux/amd64 which seemed...

OK, I accidently left that in. Just removed it. That platform line should be deleted. The idea is that the host machine drives that dynamically as described here. https://lamby.custominktech.com/docs/cpu

**Comment by mattzollinhofer** [comment]
Oh great 😀!

I just re-ran through the quick start guide and have the same failure, but I realized the problem and I'm not sure it's anything to do with you. 

When I run the first command (below) I end up with a `.devcontainers/docker-compse.yml` that is not from this cookiecutter project. I'm still too new to docker to really know if that's expected or not, but I am surprised that a pretty vanilla install of docker got be a `docker-compose.yml` with `platform: linux/amd64` set in there instead of `platform: linux/arm64`. I'm not sure if that just a me thing or not, but thought you might be interested.

```shell
docker run \
  --rm \
  --interactive \
  --volume "${PWD}:/var/task" \
  ghcr.io/customink/lamby-cookiecutter \
  "gh:customink/lamby-cookiecutter"
```


### Full Re-Run Through of Guide
Once I remove or update the `platform: linux/arm64` line I'm good to go. But on first run it errors out as I first reported. 

![CleanShot 2023-03-05 at 23 14 13@2x](https://user-images.githubusercontent.com/444766/223018399-110d4bef-2ac6-4c2c-a729-192b903f5234.png)

**Comment by mattzollinhofer** [comment]
Doesn't seem at all like something that's your issue, so I'm closing this. I wanted to share this in case this came up for others or if it triggered a thought for you.

Minor docs PR to fix a link coming your way.

**Comment by metaskills** [comment]
Reopened, here to help. There may be issues. I have a few questions.

> When I run the first command (below) I end up with a .devcontainers/docker-compse.yml that is not from this cookiecutter project.

Can you explain that? Reminder, I've been updating the cookie cuter repo with some of your feedback. Like removing platform from the compose file.

>  vanilla install of docker got be a docker-compose.yml with platform: linux/amd64 set in there instead of platform: linux/arm64.

Right, it should be removed. That line, delete it in your cookie cutter project. I feel bad... I thought I pushed a commit the removed that but I can not find it. Proof this time. https://github.com/customink/lamby-cookiecutter/commit/484816887af060babf45f198b995881c8769ff76

> Once I remove or update the platform: linux/arm64 line I'm good to go. 

Cool! Sorry about that, not sure why that 1st removal did not happen. Gonna keep this open for awhile and watch things.

**Comment by mattzollinhofer** [comment]
I think what we have here is a comedy of errors 😄. 

> Can you explain that? 

I'm pretty sure can 😆. I'm pretty sure I looked in the root level `.devcontainers` [directory](https://github.com/customink/lamby-cookiecutter/tree/master/.devcontainer) and didn't see a file called `docker-compose.yml` and (because I _barely_ understand docker) so I figured that it must be generated by `docker run` or something. Now I see that it lives in [this subdirectory](https://github.com/customink/lamby-cookiecutter/tree/master/%7B%7Bcookiecutter.project_name%7D%7D/.devcontainer).

>  I thought I pushed a commit the removed that but I can not find it

Yeah I think this got lost along the way somehow, I remember looking through the commit history and being surprised not to see this somewhere, but again, was too turned around to realize it at the time. Also, who wants to be the guy on the internet being like, are you sure you pushed??? 😂

> Gonna keep this open for awhile and watch things

It was a quick test so I just re-ran the steps locally and everything created as expected this time. Looks good! Feel free to close whenever is good for you. I'm happy with this.

> Sorry about that

What a jerk! Seriously though, I appreciate the work. I don't really if this will be a permanent part of my life, but I definitely valued the process and exposure to it. I maintained a small Vue component for a while and struggled to keep up with it, so I know there's a bunch of work that goes into.
